spec/models/ingredient_mention_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe IngredientMention, type: :model do
+  subject do
+    build(:ingredient_mention)
+  end
+
+  describe 'when observing ingredient relationship' do
+    it 'belongs to movie pairing' do
+      new_ing = Ingredient.new(name: 'Test Ingredient')
```

Comment:
please refer to these as just ingredient, movie, and ingredient_mention, instead of new_anything